awx/conf/settings.py

```diff
@@ -81,7 +81,7 @@ def _ctit_db_wrapper(trans_safe=False):
         yield
     except DBError as exc:
         if trans_safe:
-            if 'migrate' not in sys.argv and 'check_migrations' not in sys.argv:
+            if 'migrate' not in sys.argv:
```

Comment:
Both of these are checks concerning migrations.  The plan was to remove both of them.

Reply:
I have removed both the conditions.